Cookbook/Technical-Documents/UnitTestingViewModels.md

```diff
@@ -0,0 +1,51 @@
+## Unit testing view models
+
+### What do we test in the view model unit tests?
+
+- view model state changes
+- routing events
+- analytics events
```

Comment:
Should they be in a different file/test? IMO it makes it easier to read/scan.

Reply:
this can lead to unnecessary duplicated tests, as the same action can result in multiple events (state change, routing, analytics event) happening all together